pkg/query-service/app/http_handler.go

```diff
@@ -3783,7 +3783,7 @@ func (aH *APIHandler) autocompleteAggregateAttributes(w http.ResponseWriter, r *
 
 	switch req.DataSource {
 	case v3.DataSourceMetrics:
-		response, err = aH.reader.GetMetricAggregateAttributes(r.Context(), req)
+		response, err = aH.reader.GetMetricAggregateAttributes(r.Context(), req, true)
```

Comment:
should be a flag  instead of hardcoding ?

Reply:
No, we don't want to show any metrics with a dot in its name till we make sure working with dots works in all kinds of queries.